Support for runtime whitelisting (#646)
eclipse-ankaios/ankaios: agent/doc/swdesign/README.md

- impl
- utest

#### Agent allows only enabled runtimes
`swdd~agent-allows-enabled-runtimes~1`

Status: approved

> The Ankaios agent shall support enabling a specific set of runtimes through the config file, with a fallback to all available runtimes if no valid ones were configured.

Reviewer: ```suggestion
The Ankaios agent shall support enabling a specific set of runtimes through the config file, with a fallback to all available runtimes if no whitelist is provided.
```
